lib/open_project/backlogs/engine.rb

```diff
@@ -183,14 +185,17 @@ def self.settings
              type: 'Integer',
              required: false,
              show_if: -> (*) {
-               represented.project.backlogs_enabled? && represented.type.backlogs_type?
+               represented.project.backlogs_enabled? && represented.type.story?
              }
 
       schema :remaining_time,
              type: 'Duration',
              name_source: :remaining_hours,
              required: false,
-             writable: -> (*) { represented.remaining_time_writable? }
+             writable: -> (*) { represented.remaining_time_writable? },
+             show_if: -> (*) {
+               represented.project.backlogs_enabled?
+             }
```

Comment:
Does this not fit in one line? o0 Also: If it should stay multi line the style dictates that you use ``do`` and  ``end`` instead